tests/test_maya_memvid_full.py

```
@@ -0,0 +1,250 @@
+        rag_index=None,  # No FAISS
+        rag_documents=rag_documents,
+        rag_retriever=memvid_retriever,  # Using Memvid
+        api_key=api_keys["google_api_key"]
+    )
+    
+    print(f"🤖 Maya's Memvid-enhanced response: {response}")
+    
+    print("\n🗣️  Testing drink order (should use tools, not RAG)")
+    print("-" * 50)
+    
+    response_whiskey, history_after_whiskey, analysis_whiskey, rag_context_whiskey, tool_calls_whiskey = process_order(
```

Ruff (0.14.8)

Unpacked variable `analysis_whiskey` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)